spec/helpers/application_helper_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe ApplicationHelper, type: :helper do
```

Comment:
⚠️ Not sure if this is overdoing it because these methods are pretty straightforward... If it is, I can remove this file. 😸

Reply:
hahaha i appreciate the thoroughness! :)